pom.xml

```diff
@@ -75,6 +86,12 @@
             <artifactId>gson</artifactId>
             <version>2.10.1</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <!-- <version>2.3.2</version> -->
```

Comment:
If you decide to use the legacy JAXB version 2.3.2, you will need to revert the package change in CertificateUtilities.java.